easybuild/tools/filetools.py

```diff
@@ -343,10 +344,15 @@ def download_file(filename, url, path, forced=False):
     downloaded = False
     max_attempts = 3
     attempt_cnt = 0
+
+    # use blank HTTP header
+    header={'User-Agent':'EasyBuild'}
+    
     while not downloaded and attempt_cnt < max_attempts:
         try:
+            url_req = urllib2.Request(url, headers=header)
```

Comment:
@vanzod lets move this out of the while loop, only need to be constructed once, right? Also, you can merge this line with the one you addd above:

```python
url_req = urllib2.Request(url, headers={'User-Agent': 'EasyBuild'})

while ...
```